### ISSUE TITLE
Emits JSON as module instead of JSON

### ISSUE THREAD

**Issue by skylize** [webpack-contrib/file-loader#199]
As demonstrated by the minimal case below, `file-loader` re-interprets json as javascript and emits it as a module, when it should be interpreting it as json and emitting it as json.

---
Project setup
```
$ npm init -y
$ npm i -D webpack file-loader
$ mkdir src
$ echo '{"key": "value"}' > some.json
```
_webpack.config.js_
```

const path = require('path')

module.exports = {
  context: path.join(__dirname, 'src'),
  entry: {
    'some.json': './some.json',
  },
  output: {
    filename: '[name].js',
    path: path.resolve(__dirname, 'build'),
  },
  module: {
    rules: [
      {
        test: /some\.json$/,
        use: [
          'file-loader?name=some.json',
          'json-loader',
        ],
      },
    ]
  }
}

```
build 
```
$ node_modules/.bin/webpack
```
resulting files
```
build >
    some.json.js
    some.json
```
**Expected** _some.json_
```
{"key":"value"}
```
**Actual** _some.json_
```
module.exports = {"key":"value"}
```

**Comment by skylize** [comment]
note: These are the versions that installed when building this test.
```
"devDependencies": {
   "file-loader": "^0.11.2",
  "webpack": "^3.5.6"
}
```

**Comment by michael-ciniawsky** [comment]
If you are on `webpack >= v2.0.0` JSON works out-of-the-box, for `webpack =< v1.0.0` use the `json-loader` instead please

**Comment by skylize** [comment]
Okay thx, I tried skipping `json-loader` in my real project and experienced different errors. I forgot to double check if it was needed when creating this case.

After verifying `json-loader` wasn't needed, I removed it from real the project and magically the bugs I experienced previously were gone. Hahaha.

---
To clarify for anyone reading this in the future, the lesson is:
> In`webpack` v2.0+, `json-loader` will break `file-loader` if used together, but `file-loader` can handle json just fine on it's own.
